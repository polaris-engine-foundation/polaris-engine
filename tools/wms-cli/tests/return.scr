func main() {
    ret = foo();
    print(ret);

    print(bar());
}

func foo() {
    return 123;
}

func bar() {
    for(i in 0..9) {
    	 print("i=" + i);
	 if (i == 5) {
	     return i;
	 }
    }
}