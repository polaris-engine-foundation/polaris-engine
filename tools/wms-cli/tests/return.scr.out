123
i=0
i=1
i=2
i=3
i=4
i=5
5
